configure.ac

```diff
@@ -131,11 +131,22 @@ if test "x$AGENTS_LIST" = x; then
 fi
 
 if test "x$AGENTS_LIST" != xall; then
+	TMP=""
 	for j in $AGENTS_LIST; do
 		if ! test -d fence/agents/$j; then
 			AC_ERROR([Agent $j does not exists])
 		fi
+
+		if test "$j" = "lib" -o "$j" = "nss_wrapper" -o "$j" = "autodetect"; then
+			continue
+		fi
+
+		AGENT=`find $srcdir/fence/agents/$j -name '*.py' -printf '%P'`
```

Comment:
This will not work for non-python agents, e.g. fence_kdump and for directories that have two fence agents (currently only zvm/fence_zvmip and zvm/fence_zvm).